bokehjs/src/coffee/models/plots/plot_canvas.coffee

```diff
@@ -42,17 +42,21 @@ export class PlotCanvasView extends DOMView
   view_options: () -> extend({plot_view: @, parent: @}, @options)
 
   pause: () ->
-    @is_paused = true
+    if not @is_paused?
+      @is_paused = 1
+    else
+      @is_paused += 1
 
   unpause: (immediate=false) ->
-    @is_paused = false
-    if immediate
-      @render()
-    else
-      @request_render()
+    @is_paused -= 1
+    if @is_paused == 0
```

Comment:
Is there a realistic possibility for calling `unpause()` more often than `pause()`? If so, this code won't catch that. Also, later on you use a non-strict condition, which could indicate such possibility. I would keep all conditions in line, so either use `<= 0` for unpaused and `> 0` for paused, or `== 0` for unpaused and `!= 0` for paused. Also, I would rename `@is_paused` to something else and private, and use `@is_paused` for a getter, so that you don't have to repeat the condition multiple times later.